sites/example-project/src/pages/api/pagesManifest.json/+server.js

```diff
@@ -63,12 +64,14 @@ export async function GET() {
 	try {
 		const pages = {};
 
-		const pagesDir = fs.readdirSync('src/pages', { withFileTypes: true, recursive: true });
+		// for some reason `readDirSync` returns different results (on windows)??
```

Comment:
I just ran `readdir` and `readdirSync`, and they were identical (once sorted). What differences were you seeing?

Reply:
On my windows desktop I was only getting `+error.svelte` and `+layout.js`, no idea why

Reply:
I was indeed running on linux, not Windows 🙃. I'll check again.

Reply:
Okay, tried on Windows and didn't see a difference. Here's the script I wrote:

```js
import fs from 'fs'
import fsPromises from 'fs/promises'

const readdirResults = await fsPromises.readdir('src/pages', { withFileTypes: true, recursive: true })
readdirResults.sort((a, b) => a.path.localeCompare(b.path))
await fsPromises.writeFile('readdir-results.json', JSON.stringify(readdirResults, null, 2))

const readdirSyncResults = fs.readdirSync('src/pages', { withFileTypes: true, recursive: true })
readdirSyncResults.sort((a, b) => a.path.localeCompare(b.path))
fs.writeFileSync('readdir-sync-results.json', JSON.stringify(readdirSyncResults, null, 2))
```

Maybe you saw a difference due to having different options (`recursive`) and you didn't realize?

Reply:
I do prefer `fs/promises`, so we can keep that. But lets remove this comment.

```suggestion
```